content/ja/api-docs/entry/alerts.md

```diff
@@ -104,7 +104,7 @@ EditURL: https://blog.hatena.ne.jp/mackerelio/mackerelio-api-jp.hatenablog.macke
 
 ```json
 {
-  "reason": <text>
+  "reason": "<text>"
```

Comment:
Since I made tiny adjustments in the English document, I decided to reflect them to the Japanese document as well.